numerrin-interface/libpcore.cpp

```diff
@@ -0,0 +1,1041 @@
+#include <sstream>
```

Comment:
There is a script install_num_interface.sh which should make copying and sourcing .bashrc should set the LD_LIBRARY_PATH. Instructions should also be in Readme.rst. Did you try that ?